README.md

```diff
@@ -44,9 +44,12 @@ The quickest way to start a single member cluster for development purposes is to
 [Docker images](https://hub.docker.com/r/hazelcast/hazelcast/).
 
 ```bash
-docker run -p 5701:5701 hazelcast/hazelcast:4.1.1
+docker run -p 5701:5701 hazelcast/hazelcast:latest
```

Comment:
Is the `latest` tag necessary here? I thought it was the default value when you don't specify a tag. Maybe we can remove it

Reply:
done in 0e438611